{
    "schema_version": "1.6.0",
    "id": "OESA-2023-1597",
    "modified": "2023-09-09T00:00:00Z",
    "published": "2023-09-09T00:00:00Z",
    "withdrawn": "",
    "aliases": null,
    "related": [
        "CVE-2022-48565"
    ],
    "summary": "python3 security update",
    "details": "Python combines remarkable power with very clear syntax. It has modules, classes, exceptions, very high level dynamic data types, and dynamic typing. There are interfaces to many system calls and libraries, as well as to various windowing systems. New built-in modules are easily written in C or C++ (or other languages, depending on the chosen implementation). Python is also usable as an extension language for applications written in other languages that need easy-to-use scripting or automation interfaces.\r\n\r\nSecurity Fix(es):\r\n\r\nAn XML External Entity (XXE) issue was discovered in Python through 3.9.1. The plistlib module no longer accepts entity declarations in XML plist files to avoid XML vulnerabilities.(CVE-2022-48565)",
    "severity": null,
    "affected": [
        {
            "package": {
                "ecosystem": "openEuler-20.03-LTS-SP1",
                "name": "python3",
                "purl": "pkg:rpm/python3@3.7.9-33.oe1"
            },
            "severity": null,
            "ranges": [
                {
                    "type": "ECOSYSTEM",
                    "repo": "",
                    "events": [
                        {
                            "introduced": "0",
                            "fixed": "3.7.9-33.oe1",
                            "last_affected": "",
                            "limit": ""
                        }
                    ],
                    "database_specific": null
                }
            ],
            "versions": null,
            "ecosystem_specific": {
                "aarch64": [
                    "python3-3.7.9-33.oe1.aarch64.rpm",
                    "python3-debuginfo-3.7.9-33.oe1.aarch64.rpm",
                    "python3-debug-3.7.9-33.oe1.aarch64.rpm",
                    "python3-debugsource-3.7.9-33.oe1.aarch64.rpm",
                    "python3-devel-3.7.9-33.oe1.aarch64.rpm"
                ],
                "noarch": [
                    "python3-help-3.7.9-33.oe1.noarch.rpm"
                ],
                "src": [
                    "python3-3.7.9-33.oe1.src.rpm"
                ],
                "x86_64": [
                    "python3-debug-3.7.9-33.oe1.x86_64.rpm",
                    "python3-debuginfo-3.7.9-33.oe1.x86_64.rpm",
                    "python3-devel-3.7.9-33.oe1.x86_64.rpm",
                    "python3-debugsource-3.7.9-33.oe1.x86_64.rpm",
                    "python3-3.7.9-33.oe1.x86_64.rpm"
                ]
            },
            "database_specific": null
        }
    ],
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://nvd.nist.gov/vuln/detail/CVE-2022-48565"
        }
    ],
    "credits": null,
    "database_specific": {
        "severity": "Critical"
    }
}